### ISSUE TITLE
Are there any plans to allow reorder the build queue?

### ISSUE THREAD

**Issue by 304NotModified** [appveyor/ci#1808]
Are there any plans allow to reorder the build queue? 

Currently I have to cancel a build and re-add it to the queue.

**Comment by IlyaFinkelshteyn** [comment]
Did you try [builds prioritization](https://www.appveyor.com/docs/build-configuration/#builds-prioritization)?

**Comment by 304NotModified** [comment]
That's on project level, isn't it? 

I like to prioritize one branch in the same repo.

**Comment by IlyaFinkelshteyn** [comment]
I see. For now I can propose you the following workaround:
- Create separate project per branch
- Use [custom YAML](https://www.appveyor.com/docs/build-configuration/#yaml-file-alternative-naming) or UI-only configuration for each project
- [Whitelist](https://www.appveyor.com/docs/branches/#white--and-blacklisting) specific branch in specific project config (YAML or UI)
- Use project-level prioritization 

I admit that this is a bit of hack and duplication but it can unblock you now.

**Comment by IlyaFinkelshteyn** [comment]
Request from another customer email:

Given the following GitHub activity:
Open PR #10 – build starts
Open PR #11 – build queued
Merge PR #9 – build queued

Current build order:
PR #10
PR #11
Merged PR #9

Desired build order:
PR #10
Merged PR #9
PR #11

**Comment by IlyaFinkelshteyn** [comment]
Not sure we can re-order, but we can try prioritize within project (right now it is within account). What criteria makes sense? Branch? PR or not?

**Comment by IlyaFinkelshteyn** [comment]
Closing in favor of https://github.com/appveyor/ci/issues/2452
